iampolicymanagementv1/iam_policy_management_v1.go

```diff
@@ -290,7 +288,7 @@ func (iamPolicyManagement *IamPolicyManagementV1) ListPoliciesWithContext(ctx co
 //
 // Currently, only the `stringEquals` and the `stringMatch` operators are available. Resource attributes may support one
 // or both operators. For more information, see [Assigning access by using wildcard
-// policies](https://cloud.ibm.com/docs/account?topic=account-wildcard).
+// policies](https://test.cloud.ibm.com/docs/account?topic=account-wildcard).
```

Comment:
Another example of where the staging area reference appears due to the use of the "test" version of the API definition.  There are others below, but I'll avoid pointing out each one :)

Reply:
🤦